### DISCUSSION TITLE
Dexcom ONE

### DISCUSSION THREAD

**Discussion by Navid200** [Navid200/xDrip#71] [Dexcom]
https://www.dexcom.com/en-EE/shop/home?fbclid=IwAR0pLcnkxfsVHTPr3rftCVPZf7c0rBQQIWjQ0eCevHAJw9meMCeqG6aW9Dw

It is provided in specific countries only.
It provides no advantages over G6.
The corresponding official app runs on a much smaller set of phones compared to the G6 app.
It provides no follower capabilities.

xDrip, currently, supports it only in no-code mode (0000).
Entering the correct calibration code, with Dexcom ONE, will fail in xDrip.

**Comment by emou** [comment]
Hi!

Is this still the case (only no-code is supported)?

If yes, what would need to be done to add proper calibration code support?

**Comment by Navid200** [comment]
Dexcom One is now supported.
You need to use an xDrip release of October 17, 2021 or after.

**Comment by emou** [comment]
Thank you, you are awesome!
I just enabled the first ONE sensor and I confirm it is indeed working.

**Comment by Ezis22** [comment]
Hi...xDrip release of 02.12.2021.I can't find dexcom one support or need to put g6......

**Reply by Navid200** [reply]
Thanks for mentioning that.
I have added a line to the Wiki to address that for anyone who may have similar questions:
https://github.com/Navid200/xDrip/wiki/Dexcom-One

**Comment by emou** [comment]
I selected g6 and it worked. It is the same hardware, if I understand correctly.

**Reply by Ezis22** [reply]
Thanks
